python/ray/test_utils.py

```diff
@@ -547,7 +547,7 @@ def new_scheduler_enabled():
 
 
 def client_test_enabled() -> bool:
-    return os.environ.get("RAY_CLIENT_MODE") is not None
+    return os.environ.get("RAY_CLIENT_MODE", "0") == "1"
```

Comment:
Could you change it to this?
```
 def client_test_enabled() -> bool:
-    return os.environ.get("RAY_CLIENT_MODE", "0") == "1"
+    return os.environ.get(
+        "RAY_CLIENT_MODE",
+        "0") == "1" or ray._private.client_mode_hook.is_client_mode_enabled
```

Reply:
Got it. the env var is updated to 0 again. But I think just `return  ray._private.client_mode_hook.is_client_mode_enabled` should be fine?